app/models/person.rb

```diff
@@ -28,4 +28,14 @@ class Person < ApplicationRecord
   def admin?
     roles.detect(&:admin?).present?
   end
+
+  def toggle_admin!
```

Comment:
_OPINION_ 
You don't need to change this since this is just my opinion, but I want to raise the point to have the discussion.

Why is this a bang method? Does the public api provide a 'safe' alternative that you can call instead? Does this method do what it semantically says its doing.

I super don't like using bang methods unless there is an equivalent 'safe' method that the developer could be choosing instead. Otherwise I don't understand the semantics of the `!`. Almost all bang method in the Rubylang spec have an equivalent "safe" method. So the bang is semantically telling you that there is a "safer" choice.

Some people use the argument "This method is changing data", but to me that just means you need to give your method a descriptive name. `Hash#delete` doesn't take a bang and it modifies data directly. But the effect of the method is exactly what you expect from a method called delete. 
When you give your method a name that describes what it does, you dont need to add a bang to give your opinion that you think its a dangerous method.
`def my_super_scary_delete_everything_method_with_a_bang_so_everyone_knows_its_scary!` 

TL;DR I dont like using the bang to indicate your opinion that code is dangerous, that the method changes data, or that it raises an error. I only support its use to indicate that there is a safer alternative method you could be choosing